Demo_Project/src/pages/Home.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Banner from "../Component/Banner/Banner";
+import Grid from "@material-ui/core/Grid";
+import { makeStyles } from "@material-ui/core";
+import ToolBar from "../Component/ToolBar/ToolBar";
+import PageNumbers from "../Component/PageNumber/PageNumbers";
+import ArticleCard from "../Component/Artical/ArticleCard";
+import PopularTags from "../Component/PopularTags/PopularTags";
+const useStyles = makeStyles(theme => ({
+  root: {
+    flexGrow: 1
+  },
+  paper: {
+    padding: theme.spacing(2),
+    textAlign: "center",
+    color: theme.palette.text.secondary
+  },
+  right: {
+    paddingLeft: 150
+  }
+}));
+function Home({user}:{user?:any}) {
+  const flage=false
```

Comment:
what does `flage` do? it should be renamed to be more descriptive (flage isn't an english word, you probably mean flag, like every other file defined it before)